galahad/client/formats.py

```diff
@@ -73,3 +74,40 @@ def build_span_classification_document(
 
     document = Document(text=text, annotations=annotations.to_dict(), version=version)
     return document
+
+#original_doc: Document
+def build_span_classification_response(text: str, sentences: List[List[str]], spans: List[List[Span]] = None, version: int = 0) -> Document:
+    #text = original_doc.text
+    annotations = Annotations(text)
+
+    token_type = AnnotationTypes.TOKEN.value
+    span_annotation_type = AnnotationTypes.ANNOTATION.value
+    value_feature = AnnotationFeatures.VALUE.value
+
+    current = 0
+    start_lists = []
+    end_lists = []
+    for sentence in sentences:
```

Comment:
You do not need to add tokens, they should have been in the original doc already. You can write an assertion to check that and remove the `sentences` parameter.